plasmapy/dispersion/numerical/hollweg_.py

```diff
@@ -0,0 +1,285 @@
+    T_i : `~astropy.units.Quantity`
+        The ion temperature in units of :math:`K` or :math:`eV`.
+    theta : `~astropy.units.Quantity`, single valued or 1-D array
+        The angle of propagation of the wave with respect to the magnetic field,
+        :math:`\cos^{-1}(k_z / k)`, in units must be convertible to :math:`deg`.
+        Either single valued or 1-D array of size :math:`M`.
+    gamma_e : `float` or `int`, optional
+        The adiabatic index for electrons, which defaults to 1.  This
+        value assumes that the electrons are able to equalize their
+        temperature rapidly enough that the electrons are effectively
+        isothermal.
+    gamma_i : `float` or `int`, optional
+        The adiabatic index for ions, which defaults to 3. This value
+        assumes that ion motion has only one degree of freedom, namely
+        along magnetic field lines.
+    z_mean : `float` or int, optional
+        The average ionization state (arithmetic mean) of the ``ion`` composing
+        the plasma.  Will override any charge state defined by argument ``ion``.
+
+    Returns
+    -------
+    omega : Dict[str, `~astropy.units.Quantity`]
+        A dictionary of computed wave frequencies in units :math:`rad/s`.  The
+        dictionary contains three keys: ``'fast_mode'`` for the fast mode,
+        ``'alfven_mode'`` for the Alfvén mode, and ``'acoustic_mode'`` for the
+        ion-acoustic mode.  The value for each key will be a :math:`N x M` array.
+
+    Raises
+    ------
+    TypeError
+        If applicable arguments are not instances of `~astropy.units.Quantity` or
+        cannot be converted into one.
+
+    TypeError
+        If ``ion`` is not of type or convertible to `~plasmapy.particles.Particle`.
+
+    TypeError
+        If ``gamma_e``, ``gamma_i``, or``z_mean`` are not of type `int` or `float`.
+    ~astropy.units.UnitTypeError
+        If applicable arguments do not have units convertible to the expected
+        units.
+
+    ValueError
+        If any of ``B``, ``k``, ``n_i``, ``T_e``, or ``T_i`` is negative.
+
+    ValueError
+        If ``k`` is negative or zero.
+
+    ValueError
+        If ``ion`` is not of category ion or element.
+
+    ValueError
+        If ``B``, ``n_i``, ``T_e``, or ``T_I`` are not single valued
+        `astropy.units.Quantity` (i.e. an array).
+
+    ValueError
+        If ``k`` or ``theta`` are not single valued or a 1-D array.
+
+    Notes
+    -----
+    Solves the equation 3 in Bellan2012JGR (equation 38 in Hollweg1999)
+
+    .. math::
+        \left(\frac{\omega^2}{k_{\rm z}^2 v_{\rm A}^2} - 1 \right) &
+        \left[\omega^2\left(\omega^2 - k^2 v_{\rm A}^2\right) - &
+        \beta k^2 v_{\rm A}^2 \left(\omega^2 - k_{\rm z}^2 v_{\rm A}^2 \right) &
+        \right]\\ = \omega^2 \left(\omega^2 - k^2 v_{\rm A}^2 \right) k_{\rm x}^2 &
+        \left(\frac{c_{\rm s}^2}{\omega_{\rm ci}^2} - \frac{c^2}{\omega_{\rm pe}^2} &
+        \frac{\omega^2}{k_{\rm z}^2v_{\rm A}^2}\right)
```

Comment:
```suggestion
    .. math::
        \left( \frac{\omega^2}{k_{\rm z}^2 v_{\rm A}^2} - 1 \right) &
        \left[
            \omega^2 \left( \omega^2 - k^2 v_{\rm A}^2 \right)
            - \beta k^2 v_{\rm A}^2 \left(
                \omega^2 - k_{\rm z}^2 v_{\rm A}^2
            \right)
        \right] \\
        &= \omega^2 \left(\omega^2 - k^2 v_{\rm A}^2 \right) k_{\rm x}^2
        \left(
            \frac{c_{\rm s}^2}{\omega_{\rm ci}^2}
            - \frac{c^2}{\omega_{\rm pe}^2} \frac{\omega^2}{k_{\rm z}^2v_{\rm A}^2}
        \right)
```

This should get the math to render correctly in the documentation build.

Reply:
Is there a way to check what the math render looks like as I type out the text so I can check if it's correct when I do this in the future?

Reply:
You essentially have two options:

1. Write it up in a Latex editor and then copy-paste.
2. Build a local html copy of the documentation and launch it your web browser.  Nick did do a write up on this in the [documentation guide](https://docs.plasmapy.org/en/latest/development/doc_guide.html#building-documentation).  In his write up he covers using `make` and `tox` to build documentation, but, if I remember correctly, you're using windows and the `make` command does not work properly in the command prompt terminal or powershell.  To use `make` you'd need to setup and use the [wsl terminal](https://docs.microsoft.com/en-us/windows/wsl/about) (this is what I do).

If we have time, I can elaborate on this during the dispersion meeting this week.